.github/workflows/triton-benchmarks.yml

```
@@ -24,6 +24,10 @@ on:
         description: Run name
         type: string
         default: "Triton benchmarks"
+      skip_benchmarks:
+        description: List of benchmarks to skip
```

Let's make it clear it need to be a valid JSON list?
```suggestion
        description: JSON list of benchmarks to skip
```